### ISSUE TITLE
[PROBLEM] Nothing show on map

### ISSUE THREAD

**Issue by BoGnY** [OpenPoGo/OpenPoGoWeb#102]
Tomorrow, with latest master branch, the webgui don't work anymore...

This is log:

> 12:36 Loading Data..
> 12:36 Data Loaded!

As we can see, user info are not load.. Maps is set on on Luxemburg, and character is not show..

This is my console logs:

> GET http://localhost:8000/socket.io/?EIO=3&transport=polling&t=LOn5kc2 404 (File not found)   socket.io-1.4.5.js:1
> Uncaught TypeError: Cannot read property 'length' of undefined   main.js:688
> Uncaught TypeError: Cannot read property 'pokemon_family' of undefined   main.js:426
> Uncaught TypeError: Cannot read property 'length' of undefined   main.js:688

ecc.....

Commit 0e2ee3f81414ddf599740b0935b9b1c58a2a31cc is the latest that I've working..

**Comment by BobbyWibowo** [comment]
> GET http://localhost:8000/socket.io/?EIO=3&transport=polling&t=LOn5kc2 404 (File not found) socket.io-1.4.5.js:1

Have you ran Socket.IO server? From what I can tell, it's necessary.

**Comment by julienlavergne** [comment]
The problem you are having is not related to the issue on socket.io.
In main.js, loadJSON
Change `fulfill(data);` to `fulfill(JSON.parse(data));`

**Comment by BoGnY** [comment]
I don't have `fulfill` text on main.js
I use latest master branch, and loadJSON function on main.js file is:

```
loadJSON: function(path, success, error, successData) {
  $.get({
    url: path + "?" + Date.now()
  }).done(function(data) {
    if(data !== undefined) {
      success(data, successData)
    } else {
      error(data)
    }
  })
},
```

**Comment by tvb** [comment]
Having the same issue. And having no `fulfill` in main.js using latest master branch

**Comment by PotentialIngenuity** [comment]
im having the same issue. my map goes to the same location and as you can see i have all the files. but it cant find them
![image](https://cloud.githubusercontent.com/assets/6282185/17232383/54ab1188-54ed-11e6-8e36-25e139537ad9.png)

**Comment by BoGnY** [comment]
It was fixed with commit "put old loadJSON back in" ca7ab09ad013dcdd09672ea2eb64b8c57dda3bf2
Problem was loadJSON

**Comment by tvb** [comment]
I am still having the issue after pulling in ca7ab09

```
xxx - - [29/Jul/2016 11:51:53] code 404, message File not found
xxx - - [29/Jul/2016 11:51:53] "GET /socket.io/?EIO=3&transport=polling&t=LOs4JWx HTTP/1.1" 404 -
```
